cime/config/e3sm/tests.py

```diff
@@ -1,136 +1,172 @@
 # Here are the tests belonging to e3sm suites. Format is
-# <test>.<grid>.<compset>.
-# suite_name -> (inherits_from, timelimit, [test [, mods[, machines]]])
-#   To elaborate, if no mods are needed, a string representing the testname is all that is needed.
-#   If testmods are needed, a 2-ple must be provided  (test, mods)
-#   If you want to restrict the test mods to certain machines, than a 3-ple is needed (test, mods, [machines])
+# <test>.<grid>.<compset>[.<testmod>]
+#
+# suite_name : {
+#     "inherit" : (suite1, suite2, ...), # Optional. Suites to inherit tests from. Default is None. Tuple, list, or str.
+#     "time"    : "HH:MM:SS",            # Optional. Recommended upper-limit on test time.
```

Comment:
Is that upper limit per test or for the whole group?

Reply:
@rljacob , it applies as a first estimate for all the tests in the suite/group.